orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy

```diff
@@ -74,37 +74,50 @@ class DependentPipelineExecutionListener implements ExecutionListener {
       // Resolve templated pipelines if enabled.
       allPipelines = allPipelines.collect { pipeline ->
        if (V2Util.isV2Pipeline(pipeline)) {
-         return V2Util.planPipeline(contextParameterProcessor, executionPreprocessors, pipeline)
+         try {
+           return V2Util.planPipeline(contextParameterProcessor, executionPreprocessors, pipeline)
+         } catch (Exception e) {
+           log.error("Failed to plan V2 templated pipeline {}", pipeline.getOrDefault("id", "<UNKNOWN ID>"), e)
```

Comment:
should prob add a metric for this? what do you think @dreynaud ?

Reply:
thought unless it's on the dashboard noone will look at it.. 🤷‍♂

Reply:
I don't think there's necessarily a lot of value in throwing that on a dashboard. Is it a page-able offense though? If yes it deserves a metric, otherwise let's keep it in logs